src/main/java/freiman/projectile/ProjectileGraph.java

```diff
@@ -0,0 +1,29 @@
+package freiman.projectile;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class ProjectileGraph extends JComponent {
+
+    private Projectile projectile = new Projectile(0, 0);
+
+    @Override
+    protected void paintComponent(Graphics g) {
+        super.paintComponent(g);
+
+        g.translate(0, getHeight());
+        g.drawArc(0, (int) -projectile.getPeakY(),
+                (int) projectile.getInterceptX(), (int) projectile.getPeakY(),
+                0, 180);
```

Comment:
Never thought of using drawArc, but I'm not 100% sure it draws the correct curve in this situation.